spec/spec_helper.rb

```diff
@@ -9,6 +10,8 @@
   # okay
 end
 
+I18n.config.available_locales = [:en, :es, :de, :fr, :ja, :nl, :"pt-BR", :ru, :sv] 
```

Comment:
why there are so many available locales here?
but in [here](https://github.com/seejohnrun/ice_cube/pull/463/files#diff-05234c22ea23d8af294b12812e505764R5) you write only :en